### PR TITLE
--first push

### PR DESCRIPTION
**Pull request by Sanjuchilukuri**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,495 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:opsz,wght@6..12,400;6..12,500;6..12,600;6..12,700;6..12,800;6..12,900&display=swap');
+body {
```
Comment (karthik2265) [inline css/index.css]: I've noticed a set of styles that are duplicated in both empInf.css and roles.css. To enhance code organization and encourage reusability across all pages, I suggest consolidating these shared styles into a separate CSS file.

File: html/roles.html [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+                    <img src="../images/dashboard.svg" alt="dashboard image" />
+                    <p>Dashboard</p>
+                  </div>
+                </div>
+              </div>
+
+              <div class="employee-dropdown">
+                <input type="checkbox" id="click" />
+                <label for="click" class="tmp">
+                  <div class="employees">
```
Comment (karthik2265) [inline html/roles.html]: avoid writing class names matching the content
This video explains how to write semantic css take a look: https://www.youtube.com/watch?v=lWu5zf_S9R4

File: html/roles.html [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+                </ul>
+              </div>
+            </section>
+            <!-- dashboard items section ends  -->
+
+            <!-- user-management section starts -->
+
+            <section class="user-management">
+              <p>ROLE/USER MANAGEMENT</p>
+              <div class="roles-wrapper">
```
Comment (karthik2265) [inline html/roles.html]: if you notice each item in the sidebar has some common styles like padding, gap between text and icon ..etc you can have a class like sidebar-item which has these common styles and use it.

File: css/empInf.css [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+      }
+      .description {
+        margin-top: 0.5rem;
+        font-size: 1rem;
+        color: #6a6f74;
+      }
+      .btn {
+        margin-top: 0.5rem;
+        button {
+          padding: 15px 24px;
```
Comment (karthik2265) [inline css/empInf.css]: we should prefer to use rems wherever possible

**Comment by karthik2265** [comment]
Hii

**Review by karthik2265 — CHANGES_REQUESTED** [review]

**Comment by Sanjuchilukuri** [comment]
I've resolved those changes. Could you please check them? Commit ID: 309a6a9.

**Pull request merged**